cli/src/config/colors.toml

```diff
@@ -23,6 +23,8 @@
 
 "email" = "yellow"
 "username" = "yellow"
+"author" = "yellow"
+"committer" = "yellow"
```

Comment:
nit: perhaps, we can remove `"email" = ` and `"username" =` now.

And replace `"working_copy email" =` with `"working_copy author" =` and `"working_copy committer" =`? I have no idea why there's no `"working_copy username"`. That might be a bug.

Reply:
> nit: perhaps, we can remove `"email" = ` and `"username" =` now.

I'm not very experienced with the templating language. I'll take your word on it 👍. Doesn't seem make a difference in show or log.

> And replace `"working_copy email" =` with `"working_copy author" =` and `"working_copy committer" =`? I have no idea why there's no `"working_copy username"`. That might be a bug.

I'm more hesitate to make this change. If I'm understanding correctly you suggest:
```diff
-"working_copy email" = "yellow"
+"working_copy author" = "yellow"
+"working_copy committer" = "yellow"
+"working_copy username" = "yellow"
"working_copy timestamp" = "bright cyan"
```
Which is symmetric with the previous change which makes a kind of sense. However according to [this](https://martinvonz.github.io/jj/latest/templates/#signature-type), `name` , `email`, `username`, and `timestamp` are all related. Why is it then that you're suggesting that `name` and `email` be removed to but `username` be added? 

Thanks for taking the time to improve this PR 😊.

Reply:
Oh, I meant there should be `"working_copy username"` but isn't. All `"<prefix> email"` and `"<prefix> username"` pairs can be replaced with `"<prefix> author"` and `"<prefix> committer"`.
```diff
@@ -21,8 +21,8 @@
 "divergent prefix" = { fg = "red", underline = true }
 "hidden prefix" = "default"
 
-"email" = "yellow"
-"username" = "yellow"
+"author" = "yellow"
+"committer" = "yellow"
 "timestamp" = "cyan"
 "working_copies" = "green"
 "bookmark" = "magenta"
@@ -54,7 +54,8 @@
 "working_copy change_id" = "bright magenta"
 # We do not use bright yellow because of how it looks on xterm's default theme.
 # https://github.com/martinvonz/jj/issues/528
-"working_copy email" = "yellow"
+"working_copy author" = "yellow"
+"working_copy committer" = "yellow"
 "working_copy timestamp" = "bright cyan"
 "working_copy working_copies" = "bright green"
 "working_copy bookmark" = "bright magenta"
```

Reply:
Ok I made the changes as you suggested. Please let me know if I misunderstood your intentions. 

I'm working to understand the failing tests and will rebase on top of main once I do.